### DISCUSSION TITLE
MagicHome RGBW (ZJ-WFMN-B v1.1)

### DISCUSSION THREAD

**Discussion by iondarie** [RavenSystem/esp-homekit-devices#1895] [MEPLHAA Scripts]
Hello,

I recently purchased a MagicHome ZJ-WFMN-B v1.1) RGBW controller, i flashed the latest release and i used the config from the wiki:
```
{"c":{"b":[{"g":0,"t":5}]},"a":[{"t":30,"g":[15,13,12,14],"ld":[0,1,0,1],"b":[{"g":0}]}]}
```
But the light strip will not power on. Could anyone help please. 

Thank you,

**Comment by toshibochan** [comment]
I have setup color and white separate.
And I solder one button to turn on the light. Push once turn on the white, push one mere time turn off. Push twice turn on color, push twice again will turn off the color light.

{
	"c": {
		"b": [{
			"g": 0,
			"t": 5
		}]
	},
	"a": [{
		"t": 30,
		"s": 5,
		"e": 20,
		"g": [12, 5, 13],
		"b": [{
			"g": 0,
			"t": 2
		}],
		"f2": [{
			"g": 4,
			"t": 2
		}],
		"f3": [{
			"g": 3,
			"t": 2
		}]
	}, {
		"t": 30,
		"s": 5,
		"e": 20,
		"g": [15],
		"b": [{
			"g": 0
		}],
		"f2": [{
			"g": 4
		}],
		"f3": [{
			"g": 3
		}]
	}]
}
